src/status_im2/common/resources.cljs

```diff
@@ -18,7 +18,8 @@
    :keycard-logo                  (js/require "../resources/images/ui2/keycard-logo.png")
    :keycard-chip-light            (js/require "../resources/images/ui2/keycard-chip-light.png")
    :keycard-chip-dark             (js/require "../resources/images/ui2/keycard-chip-dark.png")
-   :keycard-watermark             (js/require "../resources/images/ui2/keycard-watermark.png")})
+   :keycard-watermark             (js/require "../resources/images/ui2/keycard-watermark.png")
+   :identicon-ring                (js/require "../resources/images/ui2/identicon-ring.png")})
```

Comment:
why we are hard-coding identicon ring? It should change as per user's chat key

Reply:
Its probably already implemented (@J-Son89 will know for sure), which just to need highlight it

Reply:
I got it 👍

Reply:
Yes it's there if you set a profile picture. There is a separate bug where it does not show when no profile picture is set

Reply:
sent a solution on discord which should handle this 👍

Reply:
Fixed